options.go

```diff
@@ -35,6 +35,13 @@ func WithTag(tag string, redactors ...Redactor) Option {
 	return WithCensor(newTagCensor(tag), redactors...)
 }
 
+// WithCustomTagKey is an option to set the custom tag key. The default tag key is `masq`.
+func WithCustomTagKey(tagKey string) Option {
+	return func(m *masq) {
+		m.tagKey = tagKey
+	}
+}
+
```

Comment:
```go
// WithCustomTagKey gives option to change the default
// tag key to a custom tag key
func WithCustomTagKey(key string) Option {
	return func(m *masq) {
		if key == "" {
			m.tagKey = DefaultTagKey
		}
		m.tagKey = key
	}
}
```

Reply:
Thank you for the comment. While using an empty string as a default value is one possible approach, I personally find it less intuitive and would prefer to avoid it. Instead, I have chosen to export a variable named `DefaultTagKey` for reference. [968546f](https://github.com/m-mizutani/masq/pull/19/commits/968546fd12788c5d445ec4cc6315d17b2ce76c8c)